srml/council/src/lib.rs

```diff
@@ -15,132 +15,86 @@
-//! still starts a referendum (elevating the proposal to the table of referendum) but instead applies a vote threshold
-//! of `SimplyMajority`. Lastly, if the voting tally has more yay votes than the combination of all nay votes
-//! and abstainers, then it removes any veto imposed upon the council proposal (since the proposal voting period
-//! is expiring).
-//!
-//! - **Council proposal voting approval (simply majority / majority agreement)** Upon the voting tally of yay
-//! votes for a council proposal reaching its threshold level for approval during its voting period, and where a
-//! majority council agreement occurs, whereby its tally from majority voting results in a simple majority
-//! (i.e. more explicit yay than nay votes, which signals a sensible and uncontroversial proposal), then the council
-//! proposal is approved. When executed, it is elevated to the table of active referenda on the next block,
-//! and a vote threshold of 'simple majority' is applied to the referendum.
-//!
-//! - **Council proposal voting approval (unanimous / super majority against)** If a unanimous voting tally for the
-//! council proposal occurs and results in a unanimous council agreement (i.e. only yay votes), then it is approved.
-//! When executed it is elevated to the table of active referenda on the next block, and a vote threshold of
-//! 'super majority against' is applied to the referendum. A negative
-//! [AQB](https://docs.substrate.dev/docs/glossary#section-adaptive-quorum-biasing-aqb-) is used to discourage
-//! councillors from abstaining. A single veto from a councillor cancels the proposal and prevents the agreement.
-//! Council proposals submitted this way must have over 50% approval since abstention votes will be biased in favour
-//! of the proposal (alongside any nay votes).
-//!
-//! - **Referenda** Each council or public proposal that is elevated as referenda to the table of referenda is
-//! instantly executed autonomously once its vote count reaches its threshold level for approval.
+//! - **Proposal:** A submission by a councillor. An initial vote of yay from that councillor is applied.
+//! Unlike the Democracy and Treasury modules, the `Proposal` type is very generic.
+//! - **Referendum:** The means of voting on a proposal.
+//! - **Vote:** A vote of yay or nay from a councillor on a single proposal. Councillors may change their vote.
+//! - **Veto:** A council member may veto any council proposal that exists. A vetoed proposal that's valid is set
+//! aside for a cooloff period. The vetoer cannot propose the proposal again until the veto expires.
+//! - **Vote cancellation:** At the end of a given block we cancel all referenda that have been
+//! elevated to the Table of Referenda whose voting period ends at that block and where the outcome of the vote
+//! tally was a unanimous vote to cancel the referendum.
+//! - **Voting process to elevate a proposal:** At the end of a given block we tally votes for expiring referenda.
```

Comment:
I think this poses a risk w/r/t changes in the Democracy module not being reflected in the documentation here. This is the _terminology_ section and should not describe implementation, especially implementation of a _different_ module. IMO the important part to document here is what gets decided in the Council module. Once it gets handed off to Democracy, the user should refer to those docs to see how it will be handled.

Have made some updates to try to make this better.

Reply:
Thanks i'm happy with that